pkg/controller/cassandracluster/pod.go

```diff
@@ -38,6 +38,20 @@ const (
 
 var reEndingNumber = regexp.MustCompile("[0-9]+$")
 
+// allPodContainerReady return true if all container in the Pod are ready
+func AllPodContainerReady(pod *v1.Pod) bool {
+	if pod.Status.ContainerStatuses != nil && len(pod.Status.ContainerStatuses) > 0{
```

Comment:
I think `pod.Status.ContainerStatuses` is at least set to an empty list by default ? if I'm right, we don't need to test if it's nil

Reply:
I had some panic while accessing the content of containerStatuses if it was not nil so the test seams necessary